walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/personalblog.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+    <meta charset="UTF-8">
+    <title>What Makes Me Me? </title>
```

Comment:
nit: you can remove the extra white space

Reply:
I think Julien was referring to the whitespace after "Me?"

Reply:
Okay thank you for that clarification.